### DISCUSSION TITLE
Balance Data operation not working

### DISCUSSION THREAD

**Discussion by akhilravuri-tul-scm** [vesoft-inc/nebula#4687] [Q&A] [answered]
Does Nebula Community version support Balance Data?

Once of our storaged volume got deleted and now we add a new pvc to that storaged.

Since data is in 2 storaged we want to rebalance the data so that it will be stored over 3 pods.

But, When I run Balance Data in nebula studio it returns error with `-1009:SemanticError: Data balance not support`

Thanks,
Akhil

**Comment by wey-gu** [comment, marked as answer]
> Does Nebula Community version support Balance Data?
> 
> 
> 
> Once of our storaged volume got deleted and now we add a new pvc to that storaged.
> 
> 
> 
> Since data is in 2 storaged we want to rebalance the data so that it will be stored over 3 pods.
> 
> 
> 
> But, When I run Balance Data in nebula studio it returns error with `-1009:SemanticError: Data balance not support`
> 
> 
> 
> Thanks,
> 
> Akhil

In open source version, balance function requires configuration of experimental flag being switched on. Please search for this confit in docs and configure it as true.
